src/assets/stylesheets/application.scss

```
@@ -10,6 +10,9 @@
 // Mixins
 
 // Base
+@import 'base/buttons';
+@import 'base/fonts';
+@import 'base/layouts';
```

Should we move these lines into the base layer?
```suggestion
@layer base {
  @import 'base/buttons';
  @import 'base/fonts';
  @import 'base/layouts';
}
```
You can check [here](https://tailwindcss.com/docs/adding-base-styles)

---

moved in ad19a1a 🙌🏼